packages/config-loader/src/schema/collect.ts

```diff
@@ -45,7 +45,7 @@ export async function collectConfigSchemas(
   packagePaths: string[],
 ): Promise<ConfigSchemaPackageEntry[]> {
   const schemas = new Array<ConfigSchemaPackageEntry>();
```

Comment:
Thinking `& { packageName: string }` here instead to avoid the public API change